src/lib/entities/content-type.ts

```diff
@@ -264,6 +276,9 @@ class EditorInterfaces {
     if (this._editor) {
       result.editor = this._editor
     }
+    if (this._editors) {
```

Comment:
A request with both editor and editors is invalid. So can we ensure that we only send one? I would probably lean towards ignoring `editor` if `editors` exists - but we should probably also log a warning